lib/npg_pipeline/function/haplotype_caller.pm

```diff
@@ -94,8 +96,23 @@ sub create {
         $chuncking_base_name,
         $product->chunk;
       my $output_path = $product->file_path($out_dir_path, ext => 'g.vcf.gz');
-      my $command = sprintf q{%s %s %s -I %s -O %s -L %s},
-        $self->gatk_cmd, $GATK_TOOL_NAME, $gatk_args, $input_path, $output_path, $region;
+      my $command;
+
+      if ($self->bqsr_enable($product) && $self->bqsr_apply_enable($product)) {
```

Comment:
Do we need bqsr_apply_enable?

Reply:
I am inclined to say yes. There are some projects out there producing the BQSR table but not applying it.